packages/pigeon/lib/swift_generator.dart

```diff
@@ -851,6 +854,7 @@ if (wrapped == nil) {
             }) {
               return 'try instanceManager.removeAllObjects()';
             },
+            taskQueueType: TaskQueueType.serial,
```

Comment:
is serial task queue the only supported type? do we support concurrent queue?

Reply:
![image](https://github.com/user-attachments/assets/ab3a4820-af49-4187-9cc1-52510aec1eab)

Reply:
> do we support concurrent queue?

We have never had a use case in our own plugins for concurrent queues, AFAIK nobody else has ever requested it, and I would not expect concurrent task queues to be a common use case in platform channels, so currently we have had no reason to implement it.